src/index.ts

```diff
@@ -120,7 +118,6 @@ export class GoogleStrategy<User> extends OAuth2Strategy<
 
   protected authorizationParams(): URLSearchParams {
     const params = new URLSearchParams({
-      scope: this.scope.join(GoogleStrategyScopeSeperator),
```

Comment:
this is now handled in `remix-auth-oauth2`
https://github.com/sergiodxa/remix-auth-oauth2/blob/8f5e6a44835128c088435f9a7da2c60dc7dbcf84/src/index.ts#L368